Dockerfile

```diff
@@ -3,6 +3,16 @@
 FROM ubuntu:16.04
 MAINTAINER Michael Stucki <michael@stucki.io>
 
+
+ENV \
+# ccache specifics
```

Comment:
Answering the other thread about `ENV`s.

Every line in `Dockerfile` creates a new layer and generally, it's desired to have as few layers as possible. therefore I merge those two `ENV`s into one. So at that point, I also add the `ENV`s into one single line.

> I am not a fan of using comments after escaped linebreaks. I'm surprised that this works after all...

Yeah I hate them in bash, too. But in Dockerfile, that's fully ok. Actually it's a feature of docker. See the [current ubuntu 16.04 official image dockerfile](https://github.com/tianon/docker-brew-ubuntu-core/blob/784da92cdc6ac5fc1c42e69f878f02d2b012f7b4/xenial/Dockerfile). Also I introduced the same technique in [PR37](https://github.com/bebehei/docker-lineageos/blob/6904188dbd37a7073bde00190367760ebdbaf0ad/Dockerfile#L35).